webapp/app/containers/Restore/Filters/Groups/actions.js

```diff
@@ -0,0 +1,33 @@
+//
+// Groups filter restore actions
+//
+
+import request from 'utils/request';
+import {
+  GROUPS,
+  CURRENT_GROUP,
+} from './constants';
+
+export function getCurrentGroup(currentGroup) {
+  return {
+    type: CURRENT_GROUP,
+    currentGroup,
+  };
+}
+
+export function getGroups(groups) {
+  return {
+    type: GROUPS,
+    groups,
+  };
+}
+
+export function getGroupsRequest() {
+  return function returnGetGroups(dispatch) {
+    return request
+    .get('/api/logged-in/admin/groups')
+    .end((err, res) => {
+      dispatch(getGroups(res.body.groups));
```

Comment:
From now on, check the "err" value from the answer, and redirect to login on err 401/403